### ISSUE TITLE
Question: Multiple chain reaction

### ISSUE THREAD

**Issue by daniymilner** [jhuckaby/Cronicle#147]
Hello!

Is it possible to chain a few jobs after one?

For example: One job is finished and I want to start 2 another jobs after it. What I should do?

Thanks a lot!

**Comment by jhuckaby** [comment]
Hello, and thank you for the issue report.  Currently there is no way to do this, as the Chain Reaction system can only start one job.  I will add this as a feature request, but please understand that I can't give you any ETA on when it will be implemented.  Cronicle is just a hobby project that I work on in my spare time.

Thanks again!

**Comment by frizwi** [comment]
Hi, I too am interested in this enhancement but I was thinking, is it already possible to do this now by chain reacting just the one job but one that's a shell-script which triggers multiple Cronicle jobs via its API?

**Comment by jhuckaby** [comment]
Yup, you could totally do it that way, if you wanted.  For reference, you'd need to create an [API Key](https://github.com/jhuckaby/Cronicle#api-keys), then use a command-line URL fetcher like `curl` or `wget`, to fire off the [run_event](https://github.com/jhuckaby/Cronicle#run_event) API with a HTTP GET via curl or wget.  You can pass the Event ID and API Key as query string parameters.
